src/configuration-gui/abrt-config-widget.c

```diff
@@ -120,9 +120,9 @@ static void abrt_config_widget_finalize(GObject *object);
 static AbrtAppConfiguration *
 abrt_app_configuration_new(const char *app_name)
 {
-    AbrtAppConfiguration *conf = libreport_xmalloc(sizeof(*conf));
+    AbrtAppConfiguration *conf = g_malloc(sizeof(*conf));
```

Comment:
```suggestion
    AbrtAppConfiguration *conf = g_new0(AbrtAppConfiguration, 1);
```

Reply:
Addressed in a new commit for all other instances of the same.